patches/third_party-rust-winapi-v0_3-BUILD.gn.patch

```diff
@@ -6,5 +6,5 @@ index bf081cdef85dd1c05d8f016d991cff963112de37..2c1b8df701f526bfab8d325f139197b6
    ]
    build_root = "crate/build.rs"
    build_sources = [ "crate/build.rs" ]
-+  features += [ "basetsd", "cfg", "cfgmgr32", "devpropdef", "excpt", "ktmtypes", "libloaderapi", "limits", "ntdef", "ntstatus", "processthreadsapi", "reason", "rpcndr", "timezoneapi", "vadefs", "vcruntime", "wincontypes", "wingdi", "windef", "winreg", "winuser", "wtypesbase" ]
++  features += [ "basetsd", "cfg", "cfgmgr32", "devpropdef", "excpt", "handleapi", "ktmtypes", "libloaderapi", "limits", "ntdef", "ntstatus", "processthreadsapi", "reason", "rpcndr", "timezoneapi", "vadefs", "vcruntime", "wincontypes", "wingdi", "windef", "winreg", "winuser", "wtypesbase" ]
```

Comment:
I'm confused about why an upstream change would require a change to this patch

Reply:
I think that upstream removed `handleapi` but one of our dependencies is still using it in some capacity. Maybe that means we should be adding it ourselves, but this approach was suggested here https://bravesoftware.slack.com/archives/C8ZJB9F9B/p1694024579813799?thread_ts=1693571120.228949&cid=C8ZJB9F9B  cc: @rillian

Reply:
ic, ok